var/spack/repos/builtin/packages/nvtop/package.py

```diff
@@ -7,16 +7,39 @@
 
 
 class Nvtop(CMakePackage, CudaPackage):
-    """Nvtop stands for NVidia TOP, a (h)top like task monitor for NVIDIA GPUs.
-    It can handle multiple GPUs and print information about them
-    in a htop familiar way."""
+    """Nvtop stands for Neat Videocard TOP, a (h)top like task monitor
+    for AMD and NVIDIA GPUS. It can handle multiple GPUs and print
+    information about them in a htop familiar way"""
 
     homepage = "https://github.com/Syllo/nvtop"
-    url      = "https://github.com/Syllo/nvtop/archive/1.1.0.tar.gz"
+    url      = "https://github.com/Syllo/nvtop/archive/refs/tags/2.0.1.zip"
 
-    version('1.1.0', sha256='00470cde8fc48d5a5ed7c96402607e474414d94b562b21189bdde1dbe6b1d1f3')
+    maintainers = ['marcost2']
+
+    version('2.0.1', sha256='ef18ce85d632eb1c22d3a3653976b2c088260039702df39fd0181f7cd3ae277d')
+    version('2.0.0', sha256='1651f34274c334a682f280dcb2f28d9642d44c7b22afe8c431cab91345b50f31')
+    version('1.2.2', sha256='543cbfdae3241fab1ea022402734c12e69d5988583193adaab69fdfae6e14c84')
+    version('1.2.1', sha256='197992cdd0e2e151fce91a7ba56f717e4d85b317c396001e8dbd84dc2ba363cd')
+
+    variant('nvidia', description='Support NVIDIA GPUS',
+            default=False)
+    variant('amd', description='Support AMD GPUS',
+            default=False)
```

Comment:
If you need at least one of them, you can use a multi-valued variant.

Reply:
Any specific idea for how it should be called?

Reply:
Not sure. What about `support` with values `nvidia` and `amd`:
```
nvtop support=nvidia,amd
```

Reply:
The problem with multi-valued variants is that a default needs to be set, always. So, should i default to both, one or the other or a third option that errors out as before?

Reply:
Having a default that errors out is not the best user experience :slightly_smiling_face: What are the requirements? Can we build support for both on machines that don't have the graphic cards installed? If so I'd go for that.

Reply:
Just tested this, when you go and execute it will just say "No GPU to monitor", so i guess having both as default is good